backend/sequelizeManager.js

```diff
@@ -90,31 +90,56 @@ export class SequelizeManager {
     }
 
     createConnection(configuration) {
-
-        const {
+        let {
             username, password, database, port,
             dialect, storage, host
             } = configuration;
 
-        this.log(`Creating a connection for user ${username}`, 1);
-
-        this.connection = new Sequelize(database, username, password, {
+        let options = {
             dialect,
             host,
             port,
             storage
-        });
+        };
+
+        this.log(`Creating a connection for user ${username}`, 1);
+
+        let subDialect = null;
+        if (this.connection) {
+            subDialect = this.connection.config.subDialect || null;
+        }
+        let redshiftOptions = null;
+
+        if ((dialect === 'redshift') || (subDialect === 'redshift')) {
+            subDialect = 'redshift';
+            // http://stackoverflow.com/questions/32037385/using-sequelize-with-redshift
+            Sequelize.HSTORE.types.postgres.oids.push('dummy');
+            redshiftOptions = {
+                dialect: 'postgres',
+                pool: false,
+                keepDefaultTimezone: true, // avoid SET TIMEZONE
+                databaseVersion: '8.0.2' // avoid SHOW SERVER_VERSION
+            };
+
+            options = merge(options, redshiftOptions);
```

Comment:
overwrites `{dialect: 'redshift'}` by `{dialect:'postgres'}` here before creating the connection object